.github/pull_request_template.md

```diff
@@ -0,0 +1,17 @@
+<!-- When fixing a bug: -->
+
+This PR fixes #<issue ID>.
+
+- [ ] l10n dependencies have been merged, if any.
+- [ ] I've added a unit test to test for potential regressions of this bug (or this is a front-end change, where we don't yet have unit test infrastructure).
+- [ ] Commits in this PR are minimal and [have descriptive commit messages](https://chris.beams.io/posts/git-commit/).
+
```

Comment:
```suggestion
- [ ] All UI revisions follow the [coding standards](https://github.com/mozilla/fx-private-relay/blob/main/docs/coding-standards.md)
```

Reply:
Good suggestion, added in 1cd6a666. I'm not sure what you mean by "class naming" though - you mean using the right prefixes?

And yeah, in my view a checklist is useful for things you need to manually check. Anything that can be delegated to computers should be delegated to computers :)